app/resources/linked_account_resource.rb

```diff
@@ -0,0 +1,17 @@
+class LinkedAccountResource < BaseResource
+  model_hint model: LinkedAccount::MyAnimeList
+
+  attribute :token
+  attributes :external_user_id, :share_to, :share_from, :sync_to
+  # :type is reserved for STI
+
+  has_one :user
+
+  def fetchable_fields
+    if current_user == _model
+      super
+    else
+      super - [:token]
+    end
+  end
```

Comment:
`fetchable_fields` is a method describing which attributes are visible to a given user — so, for example, only you can see your own email even though everybody can see your `User` (they just get `email: nil`)

So when this was combined previously, we had to display the row to all users (if it wasn't private) but hide the `token` field (so people can't steal your MAL password)

Now that it's split, you can hide the entire row from other users using the policy scope, instead of just hiding the `token` field with `fetchable_fields`